draft-ietf-ice-rfc5245bis.xml

```diff
@@ -2377,57 +2194,51 @@ candidate information of the peer. If the candidate does not appear
 there, then the check must have been a triggered check to a new remote
 candidate. In that case, the priority is taken as the value of the
 PRIORITY attribute in the Binding request that triggered the check
-that just completed. The pair is then added to the VALID LIST.
+that just completed. The pair is then added to the valid list.
 </t>
 </list></t>
-<t>NOTE: It will be very common that the valid pair will not be in any CHECK LIST.
-Recall that the CHECK LIST has pairs whose local candidates are never
+<t>NOTE: It will be very common that the valid pair will not be in any check list.
+Recall that the check list has pairs whose local candidates are never
 reflexive; those pairs had their local candidates converted to
 the base of the reflexive candidates, and then pruned if they
 were redundant. When the response to the Binding request arrives, the
 mapped address will be reflexive if there is a NAT between the two. In
 that case, the valid pair will have a local candidate that doesn't
-match any of the pairs in the CHECK LIST.
+match any of the pairs in the check list.
 </t>
 </section>
 
 
 <section title="Updating Candidate Pair States">
-<t>The ICE agent sets the state of both the candidate pair that generated
-the check, and the constructed valid pair, to Succeeded.
-Note that the pair which generated the check may be different than the
-valid pair constructed in <xref target="sec-valid-cons"/>.
+<t>The agent sets the states of both the candidate pair that generated
+the check and the constructed valid pair (which may be different) to
+Succeeded.
 </t>
-<t>The success of the connectivity check might also cause the state of other
-candidate pairs to change. The ICE agent MUST set the states for all other
-Frozen candidate pairs (in each CHECK LIST in the CHECK LIST SET) with
-the same foundation to Waiting.
+<t>The agent MUST set the states for all other Frozen candidate pairs
+in all check lists with the same foundation to Waiting.
 </t>
-<t>NOTE: Within a given CHECK LIST, candidate pairs with the same foundations will
+<t>NOTE: Within a given check list, candidate pairs with the same foundations will
 typically have different component ID values.
 </t>
 </section>
 
 <section title="Updating the Nominated Flag">
 
-<t>If the ICE agent was a controlling agent, and the agent had included a
-USE-CANDIDATE attribute in the Binding request, the valid pair
-generated from that check has its nominated flag value set to true. This
-flag indicates that this valid pair SHOULD be used for media, unless
-the sending agent detects that the candidate pair does not work.
-This concludes the ICE processing for this media stream or all
-media streams; see <xref target="sec-conclude"/>.
```

Comment:
I think I just removed the redundant part (what the nominated flag means is defined elsewhere), and made it shorter.